### PR TITLE
[Kotlin1] Add explanation for inheritance

### PR DESCRIPTION
**Pull request by bowserf**

### CODE REVIEW

File: mwm-3-kotlin-1/presentation/PITCHME.md [hunk 1]
```diff
@@ -546,6 +546,25 @@ public final void testForWithIndices() {
...
+```kotlin
+open class Animal(){
+	open fun myOpenedFunction() {}
+}
+class Dog() : Animal(){
+	override fun myOpenedFunction() {}
+}
+```
+
+We must use `override` language keyword to override a method.
```
Comment (Mercandj) [inline mwm-3-kotlin-1/presentation/PITCHME.md]: Smaller sentence: `override language keyword required to override a method`

File: mwm-3-kotlin-1/presentation/PITCHME.md [hunk 2]
```diff
@@ -546,6 +546,25 @@ public final void testForWithIndices() {
 
 ---
 
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Extend</span> <span style="text-transform: none; font-size:0.8em;">from a parent</span>
+
+In Kotlin, class and methods are `final` by default.
+
+Use `open` keyword in from of a class/method to be able to extend from it :
```
Comment (Mercandj) [inline mwm-3-kotlin-1/presentation/PITCHME.md]: Smaller `open keyword required to be able to extend from class/method:`

PS: "in from of" -> "in front of"

Comment (bowserf) [reply mwm-3-kotlin-1/presentation/PITCHME.md]: Fixed !

File: mwm-3-kotlin-1/presentation/PITCHME.md (hunk 1 above)
Comment (bowserf) [reply mwm-3-kotlin-1/presentation/PITCHME.md]: Fixed !

**Comment by bowserf** [comment]
thanks !

**Pull request merged**